Mirror/Runtime/LogFilter.cs

```diff
@@ -5,23 +5,6 @@ namespace Mirror
 {
     public static class LogFilter
     {
-        // this only exists for inspector UI?!
-        public enum FilterLevel
-        {
-            Developer = 0,
-            Debug = 1,
-            Info = 2,
-            Warn = 3,
-            Error = 4,
-            SetInScripting = -1
-        }
-
-        public static FilterLevel currentLogLevel = FilterLevel.Info;
-
-        internal static bool logDev { get { return currentLogLevel <= FilterLevel.Developer; } }
-        public static bool logDebug { get { return currentLogLevel <= FilterLevel.Debug; } }
-        public static bool logInfo  { get { return currentLogLevel <= FilterLevel.Info; } }
-        public static bool logWarn  { get { return currentLogLevel <= FilterLevel.Warn; } }
-        public static bool logError  { get { return currentLogLevel <= FilterLevel.Error; } }
+        public static bool logDebug = false;
```

Comment:
I would rename this to `Debug`.   The name of the class already tells you this is about logging,  so `logDebug` is redundant

Reply:
true, thanks